autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/load/YarnResponseUtils.java

```diff
@@ -23,33 +21,15 @@
 
     public List<String> getYarnRecommendedDecommissionHostsForHostGroup(String clusterCrn, YarnScalingServiceV1Response yarnResponse,
             Map<String, String> hostFqdnsToInstanceId, int maxAllowedDownScale, Optional<Integer> mandatoryDownScaleCount) {
```

Comment:
mandatoryDownScale Count will always be less than maxAllowedDownScale, but based on Yarn Workaround of passing downscaleNodeCount = clusterSize, forced downscaling should be limited to mandatoryDownScale.
So that hostGroup is not brought down to LoadAlertConfiguration:minNodeCount even though there is load on cluster.

Reply:
Added comment regarding yarnResponse might not have maxAllowedDownscale candidates